docs/v2/examples/flags.md

```diff
@@ -101,6 +101,46 @@ func main() {
 
 See full list of flags at https://pkg.go.dev/github.com/urfave/cli/v2
 
+For bool flags you can specify the flag multiple times to get a count(e.g -v -v -v or -vvv)
+
+<!-- {
+  "args": "--foo --foo",
```

Comment:
it's a bit gnarly given the problem with using `--` inside an html comment, but this appears to work for me:

```suggestion
  "args": ["&#45;&#45;foo", "&#45;&#45;foo"],
```